db/migrate/20221019091315_add_user_ref_to_like.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ] Just as suggested above this should point to the `author`, not `user`. Kindly fix that.